src/content/docs/index.md

```diff
@@ -21,8 +21,8 @@ hero:
 Hardened JavaScript is a [standards
 track](https://github.com/tc39/proposal-ses) mode for the JavaScript language
 for safe plugin systems and supply chain attack resistance.
-Hardening JavaScript improves a program’s integrity in the facy of
-adversarial code in the same process.
+Hardening JavaScript improves a program’s integrity in the face of
+adversarial code in the same Realm.
```

Comment:
technically if adversarial code runs in another non locked down realm of the same process/agent, it could measure timing. Of course the ability run adversarial code in such a realm would need to exist in the first place.

Reply:
Maybe we can link https://github.com/tc39/how-we-work/blob/main/terminology.md#realm